scripts/run-test-single.sh

```diff
@@ -0,0 +1,19 @@
+#/bin/sh
```

Comment:
This runs a single file, not a single test.
Also I'm not sure about this script is necessary.
Making it execute `npm install` everytime makes it not optimal

Reply:
test-single in package.json run à single file too, but it's named test-single.

When I develop a test, I regularly play the command "npm i && npm run single-test", I thought it would be better to have a script for that :)

Reply:
Adding a script just for a simple command might not be ideal I think

Reply:
Ok I will delete it